### PR TITLE
Pass on reordering examples

### PR DESCRIPTION
**Pull request by tinchodias**

### CODE REVIEW

**Comment by tinchodias** [comment]
Pass on ordering examples
- migrate from Brick to Bloc
- from gtExample and sampleInstance
- classify methods
- For consistency, replace DT prefix by Bl.

**Comment by tinchodias** [comment]
Ok, I missed class comments. Anyway, not convinced of the new prefix.

**Comment by Ducasse** [comment]
Excellent!

File: src/BaselineOfBlocTutorials/BaselineOfBlocTutorials.class.st [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+BaselineOfBlocTutorials >> baseline: spec [
 	<baseline>
 
 	spec for: #common do: [ 
 		spec
 			baseline: 'Memory'
 			with: [ spec repository: 'github://pharo-graphics/Tutorials:dev-1.0/src' ].
 		spec
-			baseline: 'BrickExamplesReordering'
+			baseline: 'BlocReorderingExamples'
```
Comment (cdlm) [inline src/BaselineOfBlocTutorials/BaselineOfBlocTutorials.class.st:16]: This should have been `'ReorderingExamples' to match the actual name of the baseline class

Comment (tinchodias) [reply src/BaselineOfBlocTutorials/BaselineOfBlocTutorials.class.st:16]: Oh, thanks. I'll push the fix directly to dev-1.0

Comment (tinchodias) [reply src/BaselineOfBlocTutorials/BaselineOfBlocTutorials.class.st:16]: Done. Right now there is a dependency warning that pops up later during load, but you can press Proceed. Until this is fixed.

**Pull request merged**